src/standard.yml

```diff
@@ -114,12 +114,16 @@ jobs:
       executor:
         type: executor
         default: default_executor
+      ignored_cves:
+        type: string
+        default: ""
 
     executor: << parameters.executor >>
     steps:
       - attach_workspace:
           at: ~/tmp
-      - run: bundle exec bundle-audit
+      - run: bundle exec bundle-audit update
+      - run: bundle exec bundle-audit --ignore << parameters.ignored_cves >>
```

Comment:
I am not sure if this will work properly if there are no `ignored_cves` specified. I will want to test this before we release this.

Reply:
I'm testing this here: https://github.com/tablexi/black-x/pull/523